src/routes/transactions/entities/swaps/swap-order-info.entity.ts

```diff
@@ -91,6 +92,13 @@ export class SwapOrderTransactionInfo
   })
   executedSurplusFee: string | null;
 
+  @ApiPropertyOptional({
+    type: String,
+    nullable: true,
+    description: 'The (optional) address to receive the proceeds of the trade',
+  })
```

Comment:
The Open API docs regarding the Order should be mostly the same which points to sharing these decorators. However that would require creating an abstract class with this information (which we can consider).

So far, we strived for composing the different components over inheriting them (in the Swaps feature).